tardis/model/base.py

```diff
@@ -189,6 +196,33 @@ def abundance(self):
         abundance.columns = range(len(abundance.columns))
         return abundance
 
+    def decay(self, time_explosion, normalize=True):
```

Comment:
I don't think we want a decay function at all. Maybe we need a helper function at some point but in general we don't need this.
We don't need functionality to 'redecay' a model because we will always use `time_explosion` as the time of the decay.